packages/gatsby-plugin-manifest/src/gatsby-node.js

```diff
@@ -58,8 +58,39 @@ async function checkCache(cache, icon, srcIcon, srcIconDigest, callback) {
   }
 }
 
+if (process.env.GATSBY_EXPERIMENTAL_PLUGIN_OPTION_VALIDATION) {
+  exports.pluginOptionsSchema = ({ Joi }) =>
+    Joi.object({
+      name: Joi.string(),
```

Comment:
I was wondering, shouldn't we always provide a description of the option in the schema - at least for plugins living in the monorepo?

Reply:
100% but it's kinda hard to provide descriptions for all of these as they simply correspond to the standard web app manifest options: https://web.dev/add-manifest/

I'll add them to the ones that aren't!